src/core/CMakeLists.txt

```diff
@@ -89,11 +89,11 @@ target_link_libraries(
   PRIVATE EspressoConfig shapes Profiler
           "$<$<BOOL:${FFTW3_FOUND}>:${FFTW3_LIBRARIES}>"
           "$<$<BOOL:${SCAFACOS}>:Scafacos>"
-          "$<$<BOOL:${REPA_FOUND}>:${REPA_LIBRARIES}>"
   PUBLIC
     utils MPI::MPI_CXX Random123 Boost::serialization Boost::mpi
     "$<$<BOOL:${H5MD}>:${HDF5_LIBRARIES}>"
     "$<$<BOOL:${H5MD}>:Boost::filesystem>" "$<$<BOOL:${H5MD}>:h5xx>"
+    "$<$<BOOL:${REPA_FOUND}>:${REPA_LIBRARIES}>"
```

Comment:
Just out of curiosity, why do we need to make this library public? The tests passed when it was private, with no change in code coverage, which suggests to me that the ScriptInterface and Cython shared objects didn't benefit from gaining access to it.

Reply:
@jngrad You're absolutely right, thanks. That was a misconfiguration on my side. Shouldn't have been pushed at all.